academic-initiative/jp/how-to/How-to-create-an-IBM-Cloud-account/readme.md

```diff
@@ -21,53 +23,56 @@
  
```

Comment:
Aren't we missing steps 1, 2, 3?  I also don't understand the screenshot for step4.png are we showing the Academic Initiative?

Reply:
Steps 1, 2, and 3 are same as the current page. 
Step 4 shows how to go to IBM Cloud topics page from Academic Initiative pange.